### ISSUE TITLE
Login to UF-Instant????

### ISSUE THREAD

**Issue by janner2000** [Unifi-Tools/UFiber.Configurator#24]
Why can't I login on a brand new UF-Instant SFP Module????
I tried it with SSH or Telnet.
It is also not possible to send or receive a ping to 192.168.1.1.
Did I read it correctly that you first have to flash the module via TTL???
Is there a guide to this, or a suggestion as to how to do it?

Sorry for my bad English, I'm a German user...thanks for your effort

**Comment by CurtisSWYu** [comment]
Hi, I am also new here. But did you configure the ip of your computer to 192.168.1.x subnet?

**Comment by m0veax** [comment]
Hey,

how did you tried to connect? Tell us a bit more about your Setup.

I connect to it using a tp-link MC220L and connect per telnet.

I had to start SSH there using following commands:

```
dropbear -R
```

After that, i was able to connect using ssh and copy files using scp

**Comment by DigitalPals** [comment]
Where you able to use this tool afterwards? I just got a crc error.

Furthermore I tried scp to copy files also, but couldn’t get scp to work. How did you do it?

**Comment by m0veax** [comment]
It does not work with a UFiber-Instant. I documented my thoughts in the first issue here. I‘ll drop you a link to it tomorrow

**Comment by m0veax** [comment]
https://github.com/Unifi-Tools/UFiber.Configurator/issues/1#issuecomment-1849131486

**Comment by stephenw10** [comment]
Testing this and I find that my UF-Instant is not listening on telnet or ssh. In fact it appears to only be listening on port 10001 which I assume is the Ubnt discovery service.
Is this something firmware specific? Can it be factory defaulted?
I see some references to resetting if it fails to boot 10 times. So maybe replugging it rapidly >10 times?

**Comment by m0veax** [comment]
How did you discovered the Ports? What’s your setup for connecting?

I‘m using a media converter. Which firmware version are you running?

**Comment by stephenw10** [comment]
I'm using it directly in an Intel 10G SFP NIC. It shows as linked locally without any optics connected.
I ran nmap against it from 192.168.1.100. Only 10001 shows as responding.
Attempting to telnet to it is immediately refused.
I did attempt to reset it using 10 replugs but without any idea of the expected timing I could easily have done it wrong.

**Comment by m0veax** [comment]
Hey, I remember I had to fiddle my nmap parameters to see opened ports. What did you used?

if all works out, I will check my bash_history tomorrow. It could have been -sS

**Comment by stephenw10** [comment]
I used:
```
# Nmap 7.94 scan initiated Mon Jan 15 14:17:01 2024 as: /usr/local/bin/nmap -sS -oN /root/nmap.result --append-output 192.168.1.1
Nmap scan report for 192.168.1.1
Host is up (0.0031s latency).
Not shown: 999 closed tcp ports (reset)
PORT      STATE SERVICE
10001/tcp open  scp-config
MAC Address: 74:AC:B9:99:92:xx (Ubiquiti Networks)

# Nmap done at Mon Jan 15 14:17:01 2024 -- 1 IP address (1 host up) scanned in 0.31 seconds
```
But even if nmap missed it I'd expect telnet to return a prompt on port 23.

**Comment by m0veax** [comment]
Hey,

I just booted the original and made an nmap scan:

```
# ~>  nmap -sS -p- 192.168.1.1 -T5

Starting Nmap 7.80 ( https://nmap.org ) at 2024-01-17 19:19 CET
Nmap scan report for _gateway (192.168.1.1)
Host is up (0.0033s latency).
Not shown: 65532 closed ports
PORT      STATE SERVICE
22/tcp    open  ssh
23/tcp    open  telnet
10001/tcp open  scp-config
MAC Address: XXXXXXXXXX (Unknown)
```

Could you tell me which FW Version you use?

**Comment by stephenw10** [comment]
I have no idea what firmware is running on this or how it's configured. I bought it used to experiment with. So far the experiment is not going well. ;) 
However I think I can say it isn't default since port 23 is closed.

**Comment by m0veax** [comment]
You could try to access it using UART

https://github.com/stich86/UF-Instant-Mod#uart-console-for-access-u-bootlinux-os-in-case-of-emergency

**Comment by stephenw10** [comment]
Yes, I'll do that if I'm not able to reset it. Do you know if the 10-failed-boots reset mentioned there is a thing? Or any idea what the timing might be to trigger that?

**Comment by m0veax** [comment]
I've never tried it, but it is mentioned here:

https://github.com/stich86/UF-Instant-Mod#factory-reset-after-10-boot
